<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inscription</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Broadway&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNom">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPren">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Broadway&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Prénom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Broadway&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Date Naissance</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbJ">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbM">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbA">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Broadway&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Sexe</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdoM">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Masculin</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdoF">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Féminin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Broadway&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Niveaux</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstNiveaux">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Broadway&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkA">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Allemand</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkE">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Espagnol</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkI">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Italien</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAjouter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 0, 127);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>20</width>
     <height>321</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblFiches">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>651</width>
     <height>311</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>651</width>
     <height>0</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prénom</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sexe</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date Naissance</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Classe</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Options</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="lblMsg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEng">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>350</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>save.png</normaloff>save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
